firewallns.py

```diff
@@ -0,0 +1,800 @@
+    if not template_verified:
+        return False, messages[3040]
+
+    # Generate Firewall build config
+    nftables_config = template.render(**template_data)
+
+    # Define Payloads
+    payload_create_nftables_file = f'echo "{nftables_config}" > {nftables_file}'
+    payload_validate_nftables_file = f'ip netns exec {namespace} nft --check --file {nftables_file}'
+    payload_flush_table = f'if ip netns exec {namespace} nft list tables | grep -q "inet {table}"; then' \
+                          f' ip netns exec {namespace} nft flush table inet {table}; fi'
+    payload_apply_nftables_file = f'ip netns exec {namespace} nft --file {nftables_file}'
+    payload_remvoe_nftables_file = f'rm {nftables_file}'
+
+    # Block 05: Create temp nftables.conf file on Enabled PodNet
+    # call rcc comms_ssh on enabled PodNet
+    try:
+        exit_code, stdout, stderr = comms_ssh(
+            host_ip=enabled,
+            payload=payload_create_nftables_file,
+            username='robot',
+        )
+    except CouldNotConnectException:
+        return False, messages[3050]
+
+    if exit_code != SUCCESS_CODE:
+        return False, f'{messages[3051]}\nExit Code: {exit_code}\nSTDOUT: {stdout}\nSTDERR: {stderr}'
+
+    # Block 06: Validate temp nftables.conf file on Enabled PodNet
+    try:
+        exit_code, stdout, stderr = comms_ssh(
+            host_ip=enabled,
+            payload=payload_validate_nftables_file,
+            username='robot',
+        )
+    except CouldNotConnectException:
+        return False, messages[3060]
+
+    if exit_code != SUCCESS_CODE:
+        return False, f'{messages[3061]}\nExit Code: {exit_code}\nSTDOUT: {stdout}\nSTDERR: {stderr}'
+
+    # Block 07: Flush the table if exists already on Enabled PodNet
+    try:
+        exit_code, stdout, stderr = comms_ssh(
+            host_ip=enabled,
+            payload=payload_flush_table,
+            username='robot',
+        )
+    except CouldNotConnectException:
+        return False, messages[3070]
+
+    if exit_code != SUCCESS_CODE:
+        return False, f'{messages[3071]}\nExit Code: {exit_code}\nSTDOUT: {stdout}\nSTDERR: {stderr}'
+
+    # Block 08: Apply the nftables.conf file to the namespace on Enabled PodNet
+    try:
+        exit_code, stdout, stderr = comms_ssh(
+            host_ip=enabled,
+            payload=payload_apply_nftables_file,
+            username='robot',
+        )
+    except CouldNotConnectException:
+        return False, messages[3080]
+
+    if exit_code != SUCCESS_CODE:
+        return False, f'{messages[3081]}\nExit Code: {exit_code}\nSTDOUT: {stdout}\nSTDERR: {stderr}'
+
+    # Block 09: Remove the temp nftables.conf file on Enabled PodNet
+    try:
+        remove_nftables_file, stdout, stderr = comms_ssh(
+            host_ip=enabled,
+            payload=payload_remvoe_nftables_file,
+            username='robot',
+        )
+    except CouldNotConnectException:
+        pass
```

Comment:
Why is it not being reported that the rcc.comms_ssh() could not connect to the the enabled PodNet?

Reply:
This step ie removing the temp /tmp/firewallns_<namespace>_<table>.conf is not compulsory so if it fails to connect is not a matter. So ignoring if fails to connect. But I think if a step is not compulsory then we should not use and ignore it if fails. So lets remove this step completely.